src/lib/mediumTypes.ts

```diff
@@ -0,0 +1,112 @@
+export default {
+  Architecture: {
+    id: "Architecture",
+    long_description:
+      "Includes architectural models; buildings (e.g. house, temple).",
+    name: "Architecture",
+  },
+  "Books and Portfolios": {
+    id: "Books and Portfolios",
+    long_description:
+      "Includes albums; artist books; illustrated manuscripts; portfolios (i.e. group of works such as photographs or prints).",
+    name: "Books and Portfolios",
+  },
+  "Design/Decorative Art": {
+    id: "Design/Decorative Art",
+    long_description:
+      "For objects that are primarily functional. Includes such items as containers (e.g. basket, bowl, box, vase); furniture (e.g. cabinet, chair, table); lighting; mirrors; tableware (e.g. flatware, plates).",
+    name: "Design/Decorative Art",
+  },
+  "Drawing, Collage or Other Work on Paper": {
+    id: "Design/Decorative Art",
+    long_description:
+      "For objects that are primarily functional. Includes such items as containers (e.g. basket, bowl, box, vase); furniture (e.g. cabinet, chair, table); lighting; mirrors; tableware (e.g. flatware, plates).",
+    name: "Design/Decorative Art",
```

Comment:
Seems like these didn't get updated for "Drawing, Collage or Other Work on Paper"

Reply:
Ohh, whoops! Totally right. Good catch 👍